modules/affile/affile-dest.c

```diff
@@ -325,10 +302,14 @@ affile_dw_free(LogPipe *s)
 static void
 affile_dw_notify(LogPipe *s, gint notify_code, gpointer user_data)
 {
+  AFFileDestWriter *self = (AFFileDestWriter *)s;
   switch(notify_code)
     {
     case NC_REOPEN_REQUIRED:
-      affile_dw_reopen((AFFileDestWriter *)s);
+      affile_dw_reopen(self);
+      break;
+    case NC_CLOSE:
```

Comment:
Just one note:
`NC_CLOSE` is coming through `LogWriter`, which produces the following internal message:

```
Destination timeout has elapsed, closing connection;
Destination timed out, reaping;
```

The word `connection` was a bad choice of mine, we can't talk about connections on the `LogWriter`-level, and now we actually have an implementation that is file-based.

I'd change this to something else, for example:
`Destination timeout has elapsed, closing;`, `initiating closure`, `closing fd`, `closing writer`.